core/src/main/java/nu/yona/server/subscriptions/entities/UserPrivate.java

```diff
@@ -180,4 +180,9 @@ public void touch()
 	{
 		touchVersion++;
 	}
+
+	public UUID getUserAnonymizedID()
```

Comment:
I see. In that case, ``getVPNLoginID`` should call ``getUserAnonymizedID`` (which would be faster, as currently ``getVPNLoginID`` first loads the object and then fetches its ID.